radicle-node/src/service/session.rs

```diff
@@ -154,6 +154,9 @@ pub struct Session {
     /// Last time a message was received from the peer.
     pub last_active: LocalTime,
 
+    /// Fetches queued due to another ongoing fetch.
+    pending_fetches: Vec<Id>,
```

Comment:
We can also make this one `pub(crate)` if needed. For this patch, I chose to define methods to access this field.

Reply:
Best to keep it private imo